### ISSUE TITLE
Remove interface suffix

### ISSUE THREAD

**Issue by NoUseFreak** [Cron/Cron#21]
This is something that I asked in case of many-many libraries with fear, because I don't like messing with authors personal preferences, but in most cases people welcomed my proposal. Interface suffixes is not PHP specific, unfortunately it is present in many other languages as well.

I usually link Mathias Verraes's blog post about this topic because he lists every argument against interface suffixes.

http://verraes.net/2013/09/sensible-interfaces/

My opinion is that it makes the library more readable from a programmer point of view. The lazy ones, how doesn't care about reading source code (and neither docs in most cases), just want to use it would say otherwise.

Split from #18

**Comment by NoUseFreak** [comment]
I can see how some like and some dislike the literal interface naming. 

I think removing these would break all projects currently implementing the Cron library and with that in mind I would just leave them. 

I will take your it into consideration the next time I start a project.

**Comment by sagikazarmark** [comment]
I used to use Interface suffix in my projects, here is how I dropped them:

I introduced an interface without the suffix. Moved everything to that interface and extended it with the Interface suffix version. This way you have your future interface and a deprecated one. In the next or the second major version you are free to remove the interfaces.

The point is to give enough time to users to start using the new interfaces. One or two major release should be enough.
